services/storage/src/simcore_service_storage/datcore_dsm.py

```diff
@@ -51,7 +52,12 @@ async def list_files_in_dataset(
         )
 
     async def list_files(
-        self, user_id: UserID, *, expand_dirs: bool, uuid_filter: str = ""
+        self,
+        user_id: UserID,
+        *,
+        expand_dirs: bool,
+        uuid_filter: str = "",
```

Comment:
Isn't this `uuid_filter` forgotten here? At least I can see that you removed it from other places.

Reply:
the `uuid_filter` still exists internally in the "service layer"/"business logic layer" but it not exposed as a query parameter throught the rest api